### PR TITLE
Revert "Add more detail to 2g3"

### PR DESCRIPTION
**Pull request by Rouxles**
Reverts thewca/wca-regulations#1110

### CODE REVIEW

File: wca-regulations.md [hunk 1]
```diff
@@ -77,7 +77,7 @@ Note: Because Article and Regulation numbers are not reassigned when Regulations
     - 2e3) Competitors who have no citizenship may compete as "Stateless".
 - 2f) Competitors must obey venue rules and conduct themselves in a considerate manner.
 - 2g) Competitors must remain quiet when inside the designated competition area. Talking is permitted, but must be kept at a reasonable level, and away from competitors who are actively competing.
-    - 2g3) Competitors in a group in progress must not communicate with other competitors in the group about the scrambled states of the group, unless both competitors are not eligible for any further attempts in the group. Penalty: disqualification of the competitor(s) from the event, at the discretion of the WCA Delegate. Exception: For 3x3x3 Fewest Moves, competitors that have finished their attempt may share the scramble sequence with anyone who is not actively competing in the relevant attempt.
+    - 2g3) Competitors in a Competitor Waiting Area must not communicate with each other about the scrambled states of the puzzles of the round in progress. Penalty: disqualification of the competitor(s) from the event, at the discretion of the WCA Delegate.
```
Comment (Rouxles) [inline wca-regulations.md]:
```suggestion
- 2g3) Competitors in a group in progress must not communicate with other competitors in the group about the scrambled states of the group, unless all competitors involved are not eligible for any further attempts in the group. Penalty: disqualification of the competitor(s) from the event, at the discretion of the WCA Delegate. Exception: For 3x3x3 Fewest Moves, competitors that have finished their attempt may share the scramble sequence with anyone who is not actively competing in the relevant attempt.
```

File: wca-regulations.md [hunk 2]
```diff
@@ -77,7 +77,7 @@ Note: Because Article and Regulation numbers are not reassigned when Regulations
...
-    - 2g3) Competitors in a group in progress must not communicate with other competitors in the group about the scrambled states of the group, unless both competitors are not eligible for any further attempts in the group. Penalty: disqualification of the competitor(s) from the event, at the discretion of the WCA Delegate. Exception: For 3x3x3 Fewest Moves, competitors that have finished their attempt may share the scramble sequence with anyone who is not actively competing in the relevant attempt.
+- 2g3) Competitors in a group in progress must not communicate with other competitors in the group about the scrambled states of the group, unless all competitors involved are not eligible for any further attempts in the group. Penalty: disqualification of the competitor(s) from the event, at the discretion of the WCA Delegate. Exception: For 3x3x3 Fewest Moves, competitors that have finished their attempt may share the scramble sequence with anyone who is not actively competing in the relevant attempt.
```
Comment (Rouxles) [inline wca-regulations.md]:
```suggestion
    - 2g3) Competitors in a group in progress must not communicate with other competitors in the group about the scrambled states of the group, unless all competitors involved are not eligible for any further attempts in the group. Penalty: disqualification of the competitor(s) from the event, at the discretion of the WCA Delegate. Exception: For 3x3x3 Fewest Moves, competitors that have finished their attempt may share the scramble sequence with anyone who is not actively competing in the relevant attempt.
```

File: wca-regulations.md [hunk 3]
```diff
@@ -77,7 +77,7 @@ Note: Because Article and Regulation numbers are not reassigned when Regulations
...
-    - 2g3) Competitors in a group in progress must not communicate with other competitors in the group about the scrambled states of the group, unless both competitors are not eligible for any further attempts in the group. Penalty: disqualification of the competitor(s) from the event, at the discretion of the WCA Delegate. Exception: For 3x3x3 Fewest Moves, competitors that have finished their attempt may share the scramble sequence with anyone who is not actively competing in the relevant attempt.
+    - 2g3) Competitors in a group in progress must not communicate with other competitors in the group about the scrambled states of the group, unless all competitors involved are not eligible for any further attempts in the group. Penalty: disqualification of the competitor(s) from the event, at the discretion of the WCA Delegate. Exception: For 3x3x3 Fewest Moves, competitors that have finished their attempt may share the scramble sequence with anyone who is not actively competing in the relevant attempt.
```
Comment (Nanush7) [inline wca-regulations.md]: I'm just worried that "involved" may be interpreted as "actively participating in the communication", which wouldn't include other competitors.

Comment (ohexter) [reply wca-regulations.md]: Perhaps "all such competitors" instead of "all competitors involved" then

Comment (Rouxles) [reply wca-regulations.md]:
```suggestion
    - 2g3) Competitors in a group in progress must not communicate with other competitors in the group about the scrambled states of the group, unless all such competitors are not eligible for any further attempts in the group. Penalty: disqualification of the competitor(s) from the event, at the discretion of the WCA Delegate. Exception: For 3x3x3 Fewest Moves, competitors that have finished their attempt may share the scramble sequence with anyone who is not actively competing in the relevant attempt.
```

**Review by Nanush7 — APPROVED** [review]
Let's wait for one more confirmation.

**Review by Nevseros — APPROVED** [review]
LGTM too!

**Pull request merged**